components/vocabularies/omim/api/src/main/java/org/phenotips/vocabulary/internal/solr/MendelianInheritanceInMan.java

```diff
@@ -209,14 +302,15 @@ public String getVersion()
     public synchronized int reindex(String sourceURL)
     {
         try {
-            Collection<SolrInputDocument> data = new OmimSourceParser(this.hpo, sourceURL).getData();
-            if (data.isEmpty()) {
+            index();
```

Comment:
`reindex` is supposed to start with a `clear`, otherwise stale terms will be left in the index.